README.md

```diff
@@ -27,3 +27,11 @@ To activate the plugin, create an HSM instance using:
 
     hsm create osm name Sapphire [-key=value]...
 
+The available configuration options:
+
+| Name | Description | required | default |
+| :--- | :--- | ---: | --- |
+database | The mongo database name | yes | -
+mongo_url | The mongodb connection url | yes | -
+period | The period between successive scans of flush queue | no | 1
+periodUnit | The the time unit of period, SECONDS, MINUTES ... | no | MINUTES
```

Comment:
We should use one format - camelCase or lowercase with underscore (like mongo_url)